packages/styleguide/.storybook/components/Docs/DocsContainer.tsx

```diff
@@ -26,11 +28,23 @@ const defaultComponents = {
   ...HeadersMdx,
 };
 
+/**
+ * This is not importable from SB but is ported here
+ * https://github.com/storybookjs/storybook/blob/a2752c20b83025da7b57ecb691ebeea11ab2d247/addons/docs/src/blocks/utils.ts#L41
+ */
+export function scrollToElement(element: any, block = 'start') {
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView says this is for all `Element`s.

Also, does this need to be exported?

```suggestion
function scrollToElement(element: Element, block = 'start') {
```